source/preprocessor.tex

```diff
@@ -368,6 +422,34 @@
 to \tcode{1} if the search for the source file succeeds, and
 to \tcode{0} if the search fails.
 
+\pnum
+The parenthesized \grammarterm{pp-balanced-token-seq} in each contained
+\grammarterm{has-embed-expression} is processed as if that
+\grammarterm{pp-balanced-token-seq} were the \grammarterm{pp-tokens} in the
+third form of a \tcode{\#embed} directive\iref{cpp.embed}. If such a directive
```

Comment:
Semantic line break would be nice here. I.e. prefer to put separate sentences on new lines.